reference/customers.v3.yml

```diff
@@ -4328,7 +4331,10 @@ components:
           type: integer
           format: int32
         value:
-          description: 'Attribute value. This will always be a string, regardless of the attribute''s type.'
+          description: |-
+            Attribute value. This will always be a string, regardless of the attribute's type. 
+
+            Cooresponds to `attribute_value` used in customer attribute values `GET` requests.
```

Comment:
Corresponds to `attribute_value` used in customer attribute values `GET` requests.